content/collections/browser_sdk/en/migrate-from-javascript-sdk-to-browser-sdk-2-0.md

```diff
@@ -77,6 +77,20 @@ When migrating to `@amplitude/analytics-browser@2+`, your implementation may exp
 In both versions, attribution can occur during initialization.
 {{/partial:admonition}}
 
+{{partial:admonition type="warning" heading="Cookie migration"}}
+The Browser SDK can automatically migrate cookies from JS SDK >= 6.0.0:
+
+* Old cookie name `AMP_{first 6 digit of api key}` of JS SDK >= 6.0.0
+* New cookie name `AMP_{first 10 digit of api key}` of the Browser SDK
+
+The Browser SDK does **NOT** support automatic cookie migration for JS SDK < 6.0.0.
+
+If automatic migration does not apply to your use case, you can manually migrate cookies by:
+
+1. Reading the values from the old cookie.
+2. Passing those values to `amplitude.init()` in configuration.
```

Comment:
```suggestion
2. Pass those values to `amplitude.init()` in the SDK configuration.
```

@Mercy811 Can we add a full example of what the `amplitude.init()` event looks like with the cookie values passed?

Reply:
Thanks @markzegarelli, I added it at https://github.com/amplitude/amplitude-docs/pull/579/commits/f321e94805d698ba757e7c08aeca98c21438dac4